src/components/EmptyArea.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Center, VStack, Text, HTMLChakraProps } from "@chakra-ui/react"
+import { ReactFragment } from "react"
+
+import { EmptyBoxImage } from "assets"
+
+export interface EmptyAreaOption extends HTMLChakraProps<"div"> {
+  isItemEmpty: boolean
+  actionButton: ReactFragment
+  mainText?: string
+  subText?: string
+}
+
+export const EmptyArea = ({
+  mainText,
+  subText,
+  actionButton,
+  children,
+  isItemEmpty,
+}: EmptyAreaOption): JSX.Element => {
```

Comment:
we've given this element a more powerful api by extending chakra props but the extra flexibility isn't being used - should we consider having a `...rest` argument which is spread over the `VStack`?